os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -204,18 +205,19 @@ static int is_text_address(unsigned long programCounter)
  * Below API works if there is existance of System.map file in rom fs
  ****************************************************************************/
 #ifdef CONFIG_FS_ROMFS
-void get_symbol(unsigned long search_addr, char *buffer)
+int get_symbol(unsigned long search_addr, char *buffer, size_t buflen)
 {
 	FILE *pFile;
+	int total;
 	unsigned long sym_offset;
 	unsigned long total_size;
 	unsigned long addr;
 	unsigned long next_addr;
 	int first = 0;
 	int last = 0;
 	int mid;
-	char line[128];
-	char data[6][128];
+	char line[128] = { '\0' };
+	char data[6][128] = { {'\0', '\0'} };
 	char c;
 	int n;
```

Comment:
n,i,j,k are not good names for readablity.

Reply:
kindly suggest other good names, n,i,j,k are used for referring index

Reply:
data_line? data_column? line_line? You know well than us. And What are data and line? We need to check source codes to know what it is. Please set them as we can guess easily.

Reply:
done